ote_sdk/ote_sdk/algo_backends/test_helpers/fixtures.py

```diff
@@ -0,0 +1,361 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+"""
+The file contains fixtures that may be used in algo backend's
+reallife training tests.
+
+Note that the fixtures ote_templates_root_dir_fx and ote_test_domain_fx
+MUST be overriden in algo backend's conftest.py file.
+"""
+
+# pylint: disable=redefined-outer-name
+
+import glob
+import logging
+import os
+import os.path as osp
+from copy import deepcopy
+from pprint import pformat
+from typing import Callable, Dict, Optional
+
+import pytest
+import yaml
+
+from .e2e_test_system import DataCollector
+from .training_tests_common import REALLIFE_USECASE_CONSTANT
+
+logger = logging.getLogger(__name__)
+
+#########################################################################################
+# Fixtures that should be overriden in algo backends
+
+
+@pytest.fixture(scope="session")
+def ote_templates_root_dir_fx():
+    """
+    The fixture returns an absolute path to the folder where (in the subfolders)
+    the reallife training tests will look OTE template files (the files 'template.yaml').
+
+    The fixture MUST be overriden in algo backend's conftest.py file.
+    """
+    raise NotImplementedError(
+        "The fixture ote_templates_root_dir_fx should be overriden in algo backend"
+    )
+
+
+@pytest.fixture
+def ote_test_domain_fx():
+    """
+    The fixture returns a string that will be used as the 'subject' field in the
+    e2e test system dashboard.
+    At the moment it is supposed that the fixture should return something like
+    'custom-object-detection'.
+
+    The fixture MUST be overriden in algo backend's conftest.py file.
+    """
+    raise NotImplementedError(
+        "The fixture ote_test_domain_fx should be overriden in algo backend"
+    )
+
+
+@pytest.fixture
+def ote_test_scenario_fx():
+    """
+    The fixture returns a string that will be used as the 'scenario' field in the
+    e2e test system dashboard.
+    At the moment it is supposed that the fixture should return something like
+    'api' or 'integration' or 'reallife'.
+
+    The fixture may be overriden in algo backend's conftest.py file.
+    """
+    return "api"
+
+
+#
+#########################################################################################
+
+
+def ROOT_PATH_KEY():  # pylint: disable=invalid-name
```

Comment:
follow PEP please

Reply:
In this case I will have to use usual global variables as constant, whereas I would like to use functions (to avoid changes in the variables, since python does not have `const` modificator).
Do you **really** insist on using variables instead of functions?

Reply:
Ok. Fixed